learn/configuration/instance_options.md

```diff
@@ -32,20 +32,114 @@ The above flag disables analytics for the Meilisearch instance and does not acce
 
 To configure a Meilisearch instance using environment variables, set the environment variable prior to launching the instance. If you are unsure how to do this, read more about [setting and listing environment variables](https://linuxize.com/post/how-to-set-and-list-environment-variables-in-linux/), or [use a command-line option](#command-line-options-and-flags) instead.
 
-```bash
+:::: tabs
+::: tab Linux/MacOS
```

Comment:
Would it be more accurate/inclusive to say UNIX instead of Linux? What do you think, @curquiza?

Reply:
Note to self: Update [this](https://docs.meilisearch.com/learn/security/master_api_keys.html#protecting-a-meilisearch-instance) if we go with UNIX

Reply:
Yes, in this case `Unix` is better